opencti-platform/opencti-graphql/src/database/session.js

```diff
@@ -112,14 +112,23 @@ export const killUserSessions = async (userId) => {
 };
 
 export const markSessionForRefresh = async (id) => {
+  const { store } = applicationSession;
   const currentSession = await getSession(id);
   if (currentSession) {
     const newSession = { ...currentSession, session_refresh: true };
-    await setSession(id, newSession, currentSession.expiration);
+    const sessId = id.includes(store.prefix) ? id.split(store.prefix)[1] : id;
```

Comment:
I don't understand the change

Reply:
we now use store.set which calls setSession and also sets the session in the cache, instead of setSession. And in order to call store.set we need to remove the prefix. This is what we also do when we kill a session.